torch/csrc/jit/runtime/graph_executor.cpp

```diff
@@ -799,8 +799,14 @@ void runOptimization(std::shared_ptr<Graph>& graph, bool unroll) {
 
   // Unroll small loops, and eliminate expressions that are the same at every
   // iteration.
-  if (unroll)
+  if (unroll) {
     UnrollLoops(graph);
```

Comment:
why not put loop unrolling before the peephole and constant prop that happens above?

Reply:
you want to try to get the trip count constant first